### PR TITLE
Add Design Systems

### PR DESCRIPTION
**Pull request by klaufel**
[https://github.com/klaufel/awesome-design-systems](https://github.com/klaufel/awesome-design-systems)

About design system definition, is an ever evolving collection of reusable components, guided by rules that ensure consistency and speed, by being the single source of truth for any product development, this `awesome` list, focused on frontend development of design systems. A curated list of bookmarks, articles  and more resources of design systems focused on developers.

### PR reviewed

3 pull requests I've reviewed: #1865, #1863 and #1859

https://github.com/sindresorhus/awesome/pull/1865#issuecomment-703268307
https://github.com/sindresorhus/awesome/pull/1859#issuecomment-703269398
https://github.com/sindresorhus/awesome/pull/1863#issuecomment-703268737

**[Insert URL to the list here]**

**[Explain what this list is about and why it should be included here]**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [x] Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- [x] Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- [x] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [x] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- [x] It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- [x] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- [x] The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- [x] Non-generated Markdown file in a GitHub repo.
- [x] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [x] Not a duplicate. Please search for existing submissions.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- [x] Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- [x] Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- [x] Has [contribution guidelines](awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- [x] All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- [x] Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by klaufel** [comment]
unicorn

**Comment by klaufel** [comment]
@sindresorhus I have updated the list 💪

**Comment by androiddevnotes** [comment]
@klaufel According to guidelines: Has been around for at least 30 days. That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.

You repository is not 30 days old.

**Comment by klaufel** [comment]
@androiddevnotes  Hi! The repository was created open-sourced on 20/04/2020, more than 5 months ago. 
This is one of first commits: https://github.com/klaufel/awesome-design-systems/commit/260d8211706d723cd8fb3759aa0d0872378c4cb5

**Comment by androiddevnotes** [comment]
@klaufel Thank you for clarifying.

**Comment by sdassow** [comment]
First I thought this was about design patterns for developers, but it appears to be about user interface design.
Can the explanation be more clear?

**Comment by klaufel** [comment]
> Primero pensé que se trataba de patrones de diseño para desarrolladores, pero parece que se trata de diseño de interfaz de usuario.
> ¿Puede ser más clara la explicación?

Sure @sdassow !
About design system definition, is an ever evolving collection of reusable components, guided by rules that ensure consistency and speed, by being the single source of truth for any product development, this `awesome` list, focused on frontend development of design systems.

**Comment by klaufel** [comment]
@sindresorhus it's all up to date, i don't see anything wrong right now :)

**Comment by niedev** [comment]
I analyzed your list with awesome-lint and you have to rename "Contribute" to "Contributing".

**Comment by niedev** [comment]
I've already made a pull request with the change

**Comment by niedev** [comment]
I just noticed that there is already another awesome similar to yours: https://github.com/robinstickel/awesome-design-principles#readme (search "Design Principles" in the readme, Ctrl + F) however yours seems more complete and the other has not had any changes since 2018. 
Therefore you could modify your pull request by deleting Design Principles further to add yours (I think the category where you entered your awesome is fine) so sindresorhus will be able to evaluate whether to replace it or not instead of adding a duplicate.

**Comment by horverno** [comment]
I think instead of Add design systems list should be Add Design Systems

> This pull request has a title in the format Add Name of List.
> ✅ Add Swift
> ✅ Add Software Architecture
> ❌ Update readme.md
> ❌ Add Awesome Swift
> ❌ Add swift
> ❌ add Swift
> ❌ Adding Swift
> ❌ Added Swift

**Comment by klaufel** [comment]
> I think instead of Add design systems list should be Add Design Systems
> 
> > This pull request has a title in the format Add Name of List.
> > ✅ Add Swift
> > ✅ Add Software Architecture
> > ❌ Update readme.md
> > ❌ Add Awesome Swift
> > ❌ Add swift
> > ❌ add Swift
> > ❌ Adding Swift
> > ❌ Added Swift

Update PR name, thanks @horverno :)

**Comment by klaufel** [comment]
> I just noticed that there is already another awesome similar to yours: https://github.com/robinstickel/awesome-design-principles#readme (search "Design Principles" in the readme, Ctrl + F) however yours seems more complete and the other has not had any changes since 2018.
> Therefore you could modify your pull request by deleting Design Principles further to add yours (I think the category where you entered your awesome is fine) so sindresorhus will be able to evaluate whether to replace it or not instead of adding a duplicate.

Thanks for your PR, when run the linter a few months ago it was not giving that error.

**Next steps:** Add an awesome-lint for dependency and includes on the github actions :) Thanks!

What you comment on the other awesome-list, it is focused on design principles but if it has more outdated content, we would have to think about it 🤔

**Review by damminhtien — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -285,6 +285,7 @@
 - [Material-UI](https://github.com/nadunindunil/awesome-material-ui#readme) - Material Design React components for faster and easier web development.
 - [Building Blocks for Web Apps](https://github.com/componently-com/awesome-building-blocks-for-web-apps#readme) - Standalone features to be integrated into web apps.
 - [Svelte](https://github.com/TheComputerM/awesome-svelte#readme) - App framework.
+- [Design systems](https://github.com/klaufel/awesome-design-systems#readme) - An ever evolving collection of reusable components, guided by rules that ensure consistency and speed, by being the single source of truth for any product development.
```
Comment (sindresorhus) [inline readme.md:288]: Would be great if you could shorten the description slightly.

Comment (klaufel) [reply readme.md]: Updated, now the description is on one line.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1350743496752132103

**Pull request merged**